service/src/main/kotlin/fi/espoo/evaka/invoicing/data/PaymentQueries.kt

```diff
@@ -130,6 +131,17 @@ fun Database.Read.getMaxPaymentNumber(): Long {
     return createQuery("SELECT max(number) FROM payment").exactlyOneOrNull<Long>() ?: 0
 }
 
+fun Database.Transaction.deleteDraftPayments(draftIds: List<PaymentId>) {
+    if (draftIds.isEmpty()) return
+
+    createUpdate(
+            "DELETE FROM payment WHERE status = :status::payment_status AND id = ANY(:draftIds)"
+        )
+        .bind("status", PaymentStatus.DRAFT.toString())
```

Comment:
`.toString()` shouldn't be needed here. It would also be better to use the `createUpdate<Any> { sql("""...""") }` syntax in new code.